locales/fr.json

```
@@ -54,8 +54,8 @@
     "corrupted_toml": "Fichier TOML corrompu en lecture depuis {ressource:s} (cause : {error:s})",
     "warn_the_user_about_waiting_lock": "Une autre commande YunoHost est actuellement en cours, nous attendons qu'elle se termine avant de démarrer celle là",
     "warn_the_user_about_waiting_lock_again": "Toujours en attente...",
-    "warn_the_user_that_lock_is_acquired": "l’autre commande vient de se terminer, lancement de cette commande",
+    "warn_the_user_that_lock_is_acquired": "L’autre commande vient de se terminer, lancement de cette commande",
     "invalid_token": "Jeton non valide - veuillez vous authentifier",
-    "ldap_server_is_down_restart_it": "Le service LDAP est éteint, nous tentons de le redémarrer...",
+    "ldap_server_is_down_restart_it": "Le service LDAP est arrêté, essayez de le redémarrer ...",
```

```suggestion
    "ldap_server_is_down_restart_it": "Le service LDAP est arrêté, essayez de le redémarrer...",
```
Ils n'y a pas d'espace avant les points de suspension sauf quand ils remplacent un mot.
Ex. : *Ça me fait vraiment ... de partir !*

---

Merci pour ces explications!
Par contre c'est beaucoup plus simple si les changements étaient directement fait dans weblate:
https://translate.yunohost.org/projects/yunohost/